app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>BudgetFlow</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag "https://cdnjs.cloudflare.com/ajax/libs/flowbite/1.6.6/flowbite.min.css",
+    "data-turbo-track": "reload" %>
+
+    <%= stylesheet_link_tag "tailwind", "inter-font", "data-turbo-track": "reload" %>
+
+    <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
```

Comment:
Hello @AmaduKamara 👋️, because I'm using tailwindcss, after running npm install, you have to run `rails tailwindcss:install` to bundle tailwind CSS file before running the rails server🤗️. Hence the project setup is of the following:
- `npm install`
- `rails tailwindcss:install`
- `rails server`

Also for the testing, check the test folder. I have the model test for unit tests and integration test all in there since it wasn't specified to use rspec. I went with rails default test structure